src/prognostic_equations/hyperdiffusion.jl

```diff
@@ -276,31 +277,30 @@ NVTX.@annotate function apply_tracer_hyperdiffusion_tendency!(Yₜ, Y, p, t)
     (; hyperdiff, turbconv_model) = p.atmos
     isnothing(hyperdiff) && return nothing
 
+    # When α_hyperdiff_tracer is 0, precipitating species are not hyperdiffused.
     α_hyperdiff_tracer = CAP.α_hyperdiff_tracer(p.params)
     (; ν₄_scalar_coeff) = hyperdiff
     h_space = Spaces.horizontal_space(axes(Y.c))
     h_length_scale = Spaces.node_horizontal_length_scale(h_space) # mean nodal distance
     ν₄_scalar = ν₄_scalar_coeff * h_length_scale^3
```

Comment:
```suggestion
    # Hyperdiffusivity is assumed to scale with `h^3`, where `h` is the mean nodal distance, 
    # following the empirical results of Lauritzen et al. (2018, https://doi.org/10.1029/2017MS001257)
    # Move hyperviscosity and hyperdiffusivity calculation to a function, so that we do not duplicate code (and run the risk of introducing errors, see l. 134).
    ν₄_scalar = ν₄_scalar_coeff * h_length_scale^3
```